specs/permissions.md

```diff
@@ -0,0 +1,340 @@
+
+### schemes
+The "schemes" member is a REQUIRED JSON object whose members are [Scheme objects](#schemeObject) supported by the permission. The key of each member is an identifier of the scheme and the value is a [Scheme object](#schemeObject) that contains descriptions of the permission within the scheme.   
+
+### pathSets
+The "pathSets" member is a REQUIRED JSON Array. Each element of the array is a [pathSet object](#pathSetObject). 
+
+### isHidden
+The "isHidden" member is a boolean value that indicates if a permission should be publicly usable in the API.  
+
+### requiredEnvironments
+The "requiredEnvironments" member is an array of strings that identifies the deployment environments in which the permission SHOULD be supported. When this member is not present, support for all environments is implied.
+
+### privilegeLevel
+The "privilegeLevel" member value provides a hint as to the risks of consenting this permissions. Valid values include: low, medium and high.
+
+### appIdForConditionalAccessChecks
+TBD
+
+### ownerEmail
+The "ownerEmail" member is a REQUIRED string that provides a contact mechanism for communicating with the owner of the permission. It is important that owners of permissions are aware when new paths are added to an existing permission.
+
+## <a name="pathSetObject"></a>PathSet Object
+A pathSet object identifies a set of paths that are accessible have a common set of security characteristics, such as HTTP methods and schemes. Ideally, a permission object contains a single pathSet object. This indicates that all paths protected by the permission support the same characteristics. In practice there are cases where support is not uniform. Distinct pathSet objects can be created to separate the paths with varying characteristics.  
+
+> Note: The design chosen was intentional to encourage permission creators to ensure support for methods and schemes is as consistent as possible. This produces a better developer experience for API consumers.
+
+```json
+"pathSets": [{
+        "schemeKeys": ["DelegatedWork"],
+        "methods": ["GET"],
+        "paths": {
+            "/print/settings": {}
+        }
+    },
+    {
+        "schemeKeys": ["Application"],
+        "methods": ["GET,POST"],
+        "paths": {
+            "/print/settings": {}
+        }
+    }
+] 
+```
+
+### schemeKeys
+The "schemeKeys" member is a REQUIRED array of strings that reference the schemes defined in the [permission object](#permissionObject) that are supported by the paths in this pathSet object. Each value in this array MUST match one of the keys of the "schemes" member in the [permission object](#permissionObject).
+
+### methods
+The "methods" member is a REQUIRED array of strings that represent the HTTP methods supported by the paths in this pathSet object.
+
+### paths
+The "paths" member is a REQUIRED object whose keys contain a simplified URI template to identify the resources protected by this permission object.
```

Comment:
How do we distinguish between **Read** and **Read.All** for paths permissions?

Reply:
Read and Read.All is a convention that Microsoft Graph has that indicates whether you can access any resource of the specified type, or just "yours".  It is a scoping mechanism that is opaque to this spec.  The model this spec describes currently does not have the capability to map permissions to caller identity.  At this time we are only trying to map claims to a URI template that we are primarily using to verify a user has consented an application to access data.  If you have any ideas how to describe user authZ as well, I am all ears.